qualtran/cirq_interop/_cirq_to_bloq.py

```diff
@@ -239,6 +229,39 @@ def _add_my_tensors_from_gate(
     tn.add(qtn.Tensor(data=unitary, inds=outgoing_list + incoming_list, tags=[short_name, tag]))
 
 
+def _my_tensors_from_gate(
```

Comment:
Looks like the `_add_my_tensors_from_gate` from gate is not removed but we've added ` _my_tensors_from_gate` -- is there a need to keep both?

Reply:
must have missed the removal, thanks!